tests/cnf/ran/oran/internal/helper/clusterinstance.go

```
@@ -0,0 +1,101 @@
+package helper
+
+import (
+	"fmt"
+	"os"
+
+	. "github.com/rh-ecosystem-edge/eco-gotests/tests/cnf/ran/internal/raninittools"
+	"github.com/rh-ecosystem-edge/eco-gotests/tests/cnf/ran/oran/internal/tsparams"
+	"sigs.k8s.io/yaml"
```

golangci-lint (2.12.2)

[error] import 'sigs.k8s.io/yaml' is not allowed from list 'main'

(depguard)